repl/src/main/scala/com/cloudera/livy/repl/SparkInterpreter.scala

```diff
@@ -84,12 +88,36 @@ class SparkInterpreter(conf: SparkConf) extends Interpreter {
       setContextClassLoaderMethod.invoke(sparkIMain)
 
       sparkContext = SparkContext.getOrCreate(conf)
-
+      if (conf.getBoolean("livy.repl.enableHiveContext", false)) {
```

Comment:
How do you set this?

SparkConf will only automatically load configs that start with `spark.`, and I don't see any code that's explicitly setting this config through `SparkConf.set`.